app/styles/utils/_config.scss

```diff
@@ -164,7 +164,7 @@ $color-presets: (
 
 $default-bg-color: $color-white !default;
 $secondary-bg-color: _color(bg) !default;
-$blue-bg-color: _color(bg, 200) !default;
+$blue-bg-color: _color(blue, 400) !default;
```

Comment:
Toto tu ma zostat, alebo je to len byprodukt ineho skusania?

Reply:
Toto je len letmy pokus o fix toho, aby `bg-blue` bolo realne modre. Zjavne to bolo doteraz sive. Ale mozno naschval. A ja to vlastne nepotrebujem, lebo mi to neda presne tu farbu, co potrebujem, cize to tam aj nemusi byt.

Reply:
Dal som to prec, nejdem teraz riesit.